.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+  - "2.7"
+  - "3.6"
+# command to install dependencies
```

Comment:
You shouldn't add comments unless they absolutely necessary. Either improve the make `<target>` so you don't need the comments e.g. `make dependencies` or just remove them.